cpp/CMakeLists.txt

```diff
@@ -65,6 +65,7 @@ option(USE_CCACHE "Cache build artifacts with ccache" OFF)
 option(CUDA_STATIC_RUNTIME "Statically link the CUDA runtime" OFF)
 option(CUDA_STATIC_MATH_LIBRARIES "Statically link the CUDA math libraries" OFF)
 option(CUML_USE_CUVS_STATIC "Build and statically link the CUVS library" OFF)
+option(CUML_USE_RAFT_STATIC "Build and statically link the RAFT library" OFF)
```

Comment:
So this was just defaulting to unset before but used? Weird.

Reply:
Yeah exactly. I don't see it getting set anywhere else.

```shell
git grep -E 'USE_.*_STATIC'
```